ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```
@@ -972,6 +972,14 @@ $defs:
               enabled:
                 type: bool
                 default: false
+              mode:
+                type: str
+                valid_values:
+                  - boundary
+                  - transparent
```

Since we only support boundary in `eos_designs` we should not allow other values. And especially because `transparent` is not valid :)
```suggestion
              mode:
                type: str
                valid_values:
                  - boundary
                default: boundary
```

---

Please move both of the new keys down after source_ip, since these will rarely be used.

---

Fixed!